### PR TITLE
WIP: Edit Tactics

### PR DESCRIPTION
**Pull request by yurrriq**

### CODE REVIEW

File: src/Tactics.lidr [hunk 1]
```diff
@@ -978,8 +972,10 @@ Here are some examples:
 > sillyfun_false : (n : Nat) -> sillyfun n = False
 > sillyfun_false n with (beq_nat n 3)
 >   sillyfun_false (S (S (S Z))) | True = Refl
+>   sillyfun_false n | True = Refl
 >   sillyfun_false n | False with (beq_nat n 5)
 >     sillyfun_false (S (S (S (S (S Z))))) | False | True = Refl
```
Comment (clayrat) [inline src/Tactics.lidr]: Ah, then we can simply remove both lines with `(S (S (S `

File: src/Tactics.lidr [hunk 2]
```diff
@@ -1021,6 +1017,10 @@ Now suppose that we want to convince Idris of the (rather obvious) fact that
 the proofs we did with \idr{sillyfun} above, it is natural to start the proof
 like this:
 
+\todo[inline]{Make this \idr{total}, or at least describe why it's not in its
+current implementation. Should we introduce \idr{Decidable.Equality} here?}
+
+> partial
```
Comment (clayrat) [inline src/Tactics.lidr]: Sorry, I've figured it out now:
```
sillyfun1_odd : (n : Nat) -> sillyfun1 n = True -> oddb n = True
sillyfun1_odd n prf with (beq_nat n 3) proof eq3
  sillyfun1_odd n Refl | True = 
    rewrite beq_nat_true (sym eq3) {n} {m=3} in Refl
  sillyfun1_odd n prf | False with (beq_nat n 5) proof eq5
    sillyfun1_odd n Refl | False | True = 
      rewrite beq_nat_true (sym eq5) {n} {m=5} in Refl
    sillyfun1_odd n prf | False | False = absurd prf
```

Comment (clayrat) [reply src/Tactics.lidr]: The "explicit implicits" here are unfortunate, maybe you can make it work without those.

**Comment by yurrriq** [comment]
I'm gonna go ahead and merge this now since https://github.com/idris-hackers/software-foundations/pull/21/commits/423e9a7f171b8cdadea7f6aae8eea298f319dc4d is a big deal, but the rest of this PR is fairly minor.

**Pull request merged**